tests/v1/FitnessDemo3/src/AnnotationProcessors/DAO.php

```diff
@@ -26,7 +26,7 @@ public function getReplacement(): string
             $accessors[] = $this->buildAccessors($name, $type);
         }
 
-        return implode($properties, PHP_EOL . PHP_EOL) . PHP_EOL . PHP_EOL . implode($accessors, PHP_EOL . PHP_EOL);
+        return implode(PHP_EOL . PHP_EOL, $properties) . PHP_EOL . PHP_EOL . implode(PHP_EOL . PHP_EOL, $accessors);
```

Comment:
huh, i didn't realized the "alternative" param ordering had been deprecated.
neat.